test/specs/cli/bump.spec.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { createTestProject } from '../../factories/testProject';
+import { joinAndGlob, joinAndOutputYAMLFile, runCLI, CLIButtons } from '../../utils';
+
+describe('$ bump', () => {
+
+  beforeEach(() => jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000);
+  afterEach(() => jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000);
+
+  function setup() {
+    return createTestProject();
+  }
+
+  it('moves all entries in "next"-directory to select version directory', async () => {
+    const testProject = setup();
+
+    joinAndOutputYAMLFile([testProject.changelogPath, 'next/something.yml'], {});
+
+    await runCLI(
+      testProject.rootPath,
+      ['bump'],
+      [CLIButtons.ENTER]);
```

Comment:
Also, this is implicitly selecting the first option, which happens to be the patch version, right? How about a short comment?